server/routerlicious/packages/routerlicious/config/config.json

```diff
@@ -35,6 +35,14 @@
         "kafkaClientId": "alfred",
         "maxMessageSize": "16KB",
         "maxNumberOfClientsPerDocument": 1000000,
+        "throttling": {
```

Comment:
I think we should flight this change, i.e. provide an option to disable throttling.
So if isThrottlingEnabled is true, only then Throttler will be called.

Reply:
I had this conversation with Tanvir last week, actually. We decided it would be best to default to really high values, which effectively means it will never throttle, but the idea is that http endpoints should always have throttling and that should not be optional.